### DISCUSSION TITLE
APK Problem

### DISCUSSION THREAD

**Discussion by sandeepgui98** [flet-dev/flet#4042] [Q&A]
i am making a **flet** application but how i **convert to APK**. I do so many research but **can't find** a simple way on internet. i am Window user so please tell me better way to convert **flet app to apk**.

contact no. = 8168461077

**Comment by ndonkoHenri** [comment]
- https://flet.dev/docs/publish/android
- https://youtu.be/IcT_QrLWi10
- https://youtu.be/ObO-D2TD_wo
